Sources/SwifterSwift/UIKit/UITextFieldExtensions.swift

```diff
@@ -172,6 +172,17 @@ public extension UITextField {
         rightView = iconView
         rightViewMode = .always
     }
+    
+    /// Add tool bars to the textfield input accessory view.
+    /// - Parameters:
+    ///   - items: Pass UIBarButtonItems.
+    ///   - height: Pass toolbar height (default is 44).
+    @discardableResult func addToolbar(items: [UIBarButtonItem]?, height: CGFloat = 44) -> UIToolbar {
```

Comment:
If you look in the rest of the project you'll see that the standard is to put decorations like `@discardableResult` on a separate line above the function definition.
```suggestion
    @discardableResult
    func addToolbar(items: [UIBarButtonItem]?, height: CGFloat = 44) -> UIToolbar {
```